backend/openapi.yml

```diff
@@ -0,0 +1,47 @@
+openapi: 3.0.3
```

Comment:
I've never used this before, is it just documentation?

Reply:
Yea if you've ever used like a swagger docs it seems similar. This was a cammy request so essentially before I pr actual code changes that include new endpoints I'll pr the "skeleton" which'll include the location input and output that way there should be less communication issues with what endpoints im making